third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -5614,13 +5614,13 @@ void RecordReplayEventListener::HandleRecordReplayTokenMessage(v8::Local<v8::Con
     // message is either `{ type: "connect" }` or `{ type: "login" }`, with neither payload carrying additional info.
     if (StringEquals(isolate, message_type.As<v8::String>(), "connect")) {
       LOG(ERROR) << "[RUN-2863] RecordReplayEventListener: connect message received";
```

Comment:
Do these `LOG(ERROR)` calls work as intended?

Also, why is `RecordReplayEventListener` in this file? Can we please stop adding new things into this file? 😅
(See `record_replay_events.h/cpp` for reference)

Reply:
> Do these `LOG(ERROR)` calls work as intended?

yeah, that's how we've been debugging things.  They end up in the terminal window.  We'll want to remove them at some point but they're useful to see when/what's happening.

> Also, why is `RecordReplayEventListener` in this file? Can we please stop adding new things into this file? 😅 (See `record_replay_events.h/cpp` for reference)

sure :+1: